### DISCUSSION TITLE
Does adding only issues in the project disrupt workflow?

### DISCUSSION THREAD

**Discussion by discarded-odairros** [community/community#120858] [Projects and Issues] [answered]
### Select Topic Area

Question

### Body

Hello, I would like to inquire about the functionality of Project workflows. Recently, I have only been adding issues to Projects, but I came across the Project workflow configuration, where I can set that upon approving changes in the PR, it will change the status in the project.

However, if I do not add the Pull Request to Projects and link this PR to my issue that is in Projects, when I approve the changes, will it change the status to 'Done' (as shown in the screenshot below) or will it ignore this action because the PR is not linked to the project, despite my issue linked to the PR being in the project?

![image](https://github.com/community/community/assets/137020257/52e7e793-fa01-48a6-8931-886d272959cf)

If possible, I would appreciate further clarification on this interaction. Thank you in advance for your assistance.

**Comment by Kolass2004** [comment, marked as answer]
If the Pull Request is not directly added to the Project, but it is linked to an issue that is part of the Project, the workflow configuration you mentioned may still trigger the status change to 'Done' upon approving the changes in the PR. This is because the issue linked to the PR is associated with the Project, and the workflow configuration likely operates based on the status of linked issues rather than the direct inclusion of the PR in the Project. However, it's always a good idea to test this workflow to confirm how it behaves in your specific setup.

**Reply by discarded-odairros** [reply]
Hello @Kolass2004, thank you for the response. Initially, I ran some tests and hadn't activated automation in the project's workflow. However, I've conducted further tests now, and it seems to be working correctly as you described. It moves the issue within the project as configured on the board, even if it's an action related to a PR (it works when the PR and the Issue are linked).

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @OdairRosario !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Projects and Issues category. I've gone ahead and moved it for you. Good luck!

**Reply by discarded-odairros** [reply]
Thank you @LiteBrite82! Sorry for getting the category wrong, I got a bit confused at the time. This was my first question here or anywhere else. I'll pay more attention next time.
